actors/abi/big/int.go

```diff
@@ -35,6 +35,16 @@ func PositiveFromUnsignedBytes(b []byte) Int {
 	return Int{i}
 }
 
+// MustFromString convers dec string into big integer and panics if conversion
+// is not sucessful.
+func MustFromString(s string) Int {
```

Comment:
Nit: if this is primarily for testing, I'd prefer it live in the associated test package, so it can't be used by accident in state-affecting code.

Reply:
It is also for package variables.